### PR TITLE
feat: toast

### PR DESCRIPTION
**Pull request by AmishMishra11**
image

### CODE REVIEW

**Review by amarphule — APPROVED** [review]
looks good

File: components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    <link rel="stylesheet" href="toast.css" />
+
+    <title>AM-Toast</title>
+  </head>
+
+  <body>
+    <header>
+      <nav class="desktop">
+        <div class="title">
+          <img class="logo" src="/images/Logo-modified.png" alt="" />
```
Comment (deepak29-git) [inline components/toast/toast.html]: Do not blank alt

**Review by deepak29-git — APPROVED** [review]
Rest looks good to me

File: components/toast/toast.html (hunk 1 above)
Comment (AmishMishra11) [reply components/toast/toast.html]: I've added alt

**Pull request merged**